circle.yml

```diff
@@ -41,17 +41,6 @@ test:
     # behind a user namespace which Circle CI tests are run under
     - sudo env "PATH=$PATH" bin/yarn install --force
 
-deployment:
-  release:
-    tag: /v[0-9]+(\.[0-9]+)*/
-    owner: yarnpkg
-    commands:
-      # Only NPM is handled here - All other release files are handled in a webhook.
-      - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
-      - ./scripts/set-installation-method.js "`pwd`/package.json" npm
-      # https://github.com/yarnpkg/yarn/issues/610, need to make yarn publish noninteractive
-      - npm publish --tag rc
```

Comment:
This will now be ran by the Jenkins job. To clarify the overall flow, it's:

1. Someone tags the new release
2. CircleCI and AppVeyor build the release and then call their webhooks
3. Once both the CircleCI and AppVeyor builds are both completed, the webhook code kicks off the `yarn-version` Jenkins build job
4. `yarn-version` updates the version number on the Yarn site, then kicks off `yarn-chocolatey`, `yarn-homebrew` and `yarn-npm` build jobs
5. `yarn-npm` pushes to npm

(I'll update the docs at some point)